.github/pull_request_template.md

```diff
@@ -36,6 +36,7 @@ Related changes
 If no, just leave the release note block below as is.
 If yes, a release note is required:
 Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
+Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/docs/release-note-guide.md) before writing the release note.
```

Comment:
If there is no user facing changes, should contributors to leave it as `NONE` or just remove it?

Reply:
Documented in the above: "If no, just leave the release note block below as is."